_posts/blog/2019-08-19-Ruslan's-Final-Report.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: GSoC Final Report by Ruslan 
+categories: blog
+excerpt: work product submission on the scona project during GSoC program
+image:
+  feature:
+link:
+date: 2019-08-19
+modified:
+share: true
+author: ruslan_yermakov
+---
+
+## TL;DR - GSoC Summary
+
+> **Student**: Ruslan Yermakov  
+**Mentor**: Dr Kirstie Whitaker    
+**Organization**: [INCF](https://www.incf.org/gsoc2019/projectlist)    
+**Project**: In the course of the program I was working on the open-source project [**scona**](https://github.com/WhitakerLab/scona) - a toolkit to perform brain network analyses. 
+The goal of my project was to produce publication-ready brain network analysis results and visualizations from the command line. 
+
+In this blog I am sharing the achieved deliverables and results with the community.
+
+## The scope of the project
+
+Neuroscientists and neuroimaging researches perform structural network analyses on the MRI data.
+Scona is designed to imrove this experience and to help researchers run brain analyses quickly and reliably in order to capture many aspects of brain structure and function.
+
+My project is mainly focused on the visualisation part that is now included in the scona package.
+  
+Achieved deliverables:
+- implementation of visualization module to plot MRI data and the results of structural covariance brain network analyses;  
+- development of visualisation functions which allows users to produce publication-ready plots based on the results of the performed analysis for publication in a scientific journal;  
```

Comment:
Make sure you're consistent between British and US spellings of visualisation (vs visualization)! Same for colour and color. I'm not sure I've been in the tutorials but it is important to catch. I don't mind which one you go for - pick your favourite and stick with it 😄

Reply:
Over this time I got used to the British spelling - "visualisation" :laughing: